backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/utils/MapCsvToStore.kt

```diff
@@ -0,0 +1,11 @@
+package app.ehrenamtskarte.backend.stores.utils
+
+import app.ehrenamtskarte.backend.stores.COUNTRY_CODE
+import app.ehrenamtskarte.backend.stores.importer.common.types.AcceptingStore
+import app.ehrenamtskarte.backend.stores.webservice.schema.types.CSVAcceptingStore
+
+fun mapCsvToStore(csvStore: CSVAcceptingStore): AcceptingStore {
+    return AcceptingStore(
```

Comment:
if some fields are null, they are inserted as empty in the database.
for example, the first 2 records were inserted from CSV and the rest were inserted the old way:
![image](https://github.com/user-attachments/assets/c9cb61b8-9775-4d78-b301-9faf50748169)
I assume null-values are expected in such cases.